packages/app-config-writer/src/types/variantsConfig.ts

```diff
@@ -0,0 +1,16 @@
+import type { MiddlewareConfig as PreviewConfig } from '@sap-ux/preview-middleware';
+
+export type FioriToolsDeprecatedPreviewConfig = {
```

Comment:
@tobiasqueck There already exists a type that represents the deprecated config preview config in [ui5-config/src/types/index.ts](https://github.com/SAP/open-ux-tools/blob/main/packages/ui5-config/src/types/index.ts#L49). That still seems to be used to generate outdated configs (see [here](https://github.com/SAP/open-ux-tools/blob/main/packages/ui5-config/src/ui5config.ts#L238)). Do we need to address this in this PR as well? Or is this part of some generator adjustments that will come later because it seems to be used in app writer?

Reply:
Should this type (and previewConfigOptions) be moved to preview middleware?

Reply:
I think the conflict/duplication between the types in `ui5-config` and the corresponding middlewares is a separate bigger problem. I don't have a great idea to solve it but I think it needs to be solved just not here. So, here I am ok with having duplicated definitions of the type.